.circleci/config.yml

```diff
@@ -152,7 +146,10 @@ jobs:
                 cp unpacked_sdist/.coverage .coverage
                 codecov --required
       - unless:
-          condition: << parameters.codecov >>
+          condition:
+            or:
+              - equal: [ "3.7", << parameters.image_tag >> ]
+              - equal: [ "3.8", << parameters.image_tag >> ]
```

Comment:
I think we want to use the same condition we use above for `when`. My understanding is that `when` gets run if the condition is true, and `unless` gets run when the condition is false: https://circleci.com/docs/2.0/reusing-config/#the-when-step

Reply:
Gotcha, fixed